cmake/Modules/Packages/KOKKOS.cmake

```diff
@@ -31,6 +31,10 @@ elseif(EXTERNAL_KOKKOS)
 else()
   set(LAMMPS_LIB_KOKKOS_SRC_DIR ${LAMMPS_LIB_SOURCE_DIR}/kokkos)
   set(LAMMPS_LIB_KOKKOS_BIN_DIR ${LAMMPS_LIB_BINARY_DIR}/kokkos)
+  # enforce using nvcc_wrapper as compiler wrapper
```

Comment:
@junghans is this correct?
Any idea how to "find" the nvcc_wrapper script for the other 2 cases?
It would be great, if we would not have to document how to set the wrapper and that we would override any other input.
There probably should be a warning message about the override, too.

Reply:
We could use:
```
find_program(NVCC_WRAPPER_EXECUTABLE nvcc_wrapper)
find_package_handle_standard_args(NVCC_WRAPPER REQUIRED_VARS NVCC_WRAPPER_EXECUTABLE)
```

Reply:
i've decided to save this for later. I'm including the option for the bundled Kokkos distribution in the preset (which should also work for other Kokkos installations for as long as we bundle it, we might just keep only the compiler wrapper in case we unbundle Kokkos).

Reply:
Sounds good, we can always make it as `file(DOWNLOAD ...)` when we unbundle kokkos.